README.md

```diff
@@ -20,3 +27,9 @@ You can run the previously built image with:
 ```
 docker run -it cyberbotics/webots:latest /bin/bash
 ```
+
+## Clean the temporary Images
+You can run the following command to remove **all** temporary images:
+```
+docker system prune -f
```

Comment:
```suggestion
docker system prune
```
I would remove the --force option, so that the user can see the warning and be aware that this could also affect other images not related to Webots.